e2e-tests/commands/insurance/complete-prepare-application-section-multiple-policy-type.js

```diff
@@ -32,9 +31,16 @@ const completePrepareApplicationMultiplePolicyType = ({
   // submit "policy and exports - check your answers" form
   submitButton().click();
 
-  // start "your business" section
+  // start "export contract" section
   startNowLink().click();
 
+  cy.completeAndSubmitAboutGoodsOrServicesForm({});
+
+  // submit "export contract - check your answers" form
+  submitButton().click();
```

Comment:
could we not use completeExportContractSection here?

Reply:
This whole file / the files in `complete-prepare-application-section` need a clean up/refactor - want to clean up separately after this PR

Reply:
`completeExportContractSection` actually "starts" the section via the task list, as well as complete all forms (currently only 1 form),

In this test, the flow is "at the end of X section - redirect to the start of the next section", rather than the task list which `completeExportContractSection` and other commands expect